cmd/relationstemplate.go

```diff
@@ -87,11 +87,11 @@ func setContainerTemplateRelation(c *lxd.Client, container string, tmpl string,
 
 		err := templates.parse()
 		if err != nil {
-			return errors.Wrap(err, "Failed parse data")
+			return errors.Wrap(err, "Failed parse yaml data")
 		}
 
 		if templates.Templates == nil {
-			return fmt.Errorf(i18n.G("Error no image entry here, maybe something went wrong?"))
+			return fmt.Errorf(i18n.G("Error no LXC Image entry here, maybe something went wrong?"))
```

Comment:
what happens if the template creation fails, then there is no entry here, but a container is still running. devlx is at he moment not able to remove that pending container. Should I imporve the error message or try to force the container? Suggestions would be appriciated